views/partials/requestData.pug

```diff
@@ -0,0 +1,157 @@
+h1(id='request-data-title') Destiny Hunter API
+div(class='request-data' id='request-data-1')
+  div(class='section')
+    div(class='label') Method:
+    div(class='value') GET
+  div(class='section')
+    div(class='label') Route:
+    div(class='value') /hunters
+  div(class='section')
+    div(class='label') Description:
+    div(class='value') Returns a list of all hunters in Destiny.
+
+div(class='request-data hidden' id='request-data-2')
+  div(class='section')
+    div(class='label') Method:
+    div(class='value') GET
+  div(class='section')
+    div(class='label') Route:
+    div(class='value') /hunter/{tag}
+  div(class='section')
+    div(class='label') Description:
+    div(class='value') Returns the hunter with the tag provided in the route.
+
+div(class='request-data hidden' id='request-data-3')
+  div(class='section')
+    div(class='label') Method:
+    div(class='value') GET
+  div(class='section')
+    div(class='label') Route:
+    div(class='value') /hunter/{subclass}
+  div(class='section')
+    div(class='label') Description:
+    div(class='value') Returns a list of hunters in Destiny with the subclass provided in the route.
+
+div(class='request-data hidden' id='request-data-4')
+  div(class='section')
+    div(class='label') Method:
+    div(class='value') POST
+  div(class='section')
+    div(class='label') Route:
+    div(class='value') /hunters
+  div(class='section')
+    div(class='label') Header:
+    div(class='value') Content-Type: application/json
+  div(class='section')
+    div(class='label') Body:
+    div(class='value')  {"tag": "zettlechips", "subclass": "Arcstrider", "grenades": "Skipbolt", "tree": "Way of the Wind", "exotic": "Young Ahamkaras Spine"}
+  div(class='section')
+    div(class='label') Description:
+    div(class='value') Creates a new hunter from the data provided in the body.
+
+div(class='request-data hidden' id='request-data-5')
+  div(class='section')
+    div(class='label') Method:
+    div(class='value') DELETE
+  div(class='section')
+    div(class='label') Route:
+    div(class='value') /hunters/{tag}
+  div(class='section')
+    div(class='label') Description:
+    div(class='value') Deletes the hunter associated with the tag provided in the route.
```

Comment:
Are tags unique to one hunter? I would expect tags to apply to many hunters and therefore using it for a delete is dangerous since you could wipe out lots of information. Generally you want a unique ID for a delete.